app/customer.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            money: int,
+            car: Car,
+            product_cart: dict,
+            customer_location: list

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]